src/main/java/org/mule/extension/api/runtime/FixedContextMetadata.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.extension.api.runtime;
+
+/**
+ * Implementation of {@link ContentMetadata} which does not allow changing
+ * the output metadata. Therefore, this class is immutable.
+ *
+ * @since 1.0
+ */
+public final class FixedContextMetadata implements ContentMetadata
```

Comment:
This is contains the contentType of the "payload" that enters and leaves an operation. How is the contentType of properties manager with this schema?